tools/autograd/gen_python_functions.py

```diff
@@ -95,7 +95,7 @@
     'nonzero(_(out|numpy))?',
     'set_data',
     '.*_overrideable',  # overrideable functions for backend extension
-    'data', 'is_leaf', 'output_nr', '_version', 'requires_grad_', 'retain_grad', 'set_',
```

Comment:
Its removed from this list so that the python bindings would be generated, as its a native function. I'm guessing it was here previously because we already had the duplicate python version